scripts/launch_validator_node.sh

```diff
@@ -83,9 +83,18 @@ function become_a_validator {
 
 function launch_nodes {
   define_config_file $NETWORK
+
+  # start validator node
   start_validator_node
-  echo "NODE_NAME=$NODE_NAME" >> configs/${CONFIG_FILE}
+
+  sed -i "s|NODE_NAME=.*|NODE_NAME=$NODE_NAME|" configs/${CONFIG_FILE}
+
+  # update scripts/add-validator/.env
+  update_configs
```

Comment:
Rename to `update_add_validator_configs`

Reply:
updated